### ISSUE TITLE
TEST: Add AuthSpecs

### ISSUE THREAD

**Issue by fsanaulla** [fsanaulla/chronicler#44]
After adding `AuthSpec` to akka-http module. It starts failing with this error:
```
## Exception when compiling 22 sources to IdeaProjects\chronicler\akka-http\target\scala-2.12\test-classes
null
scala.reflect.internal.Trees.traverseComponents$1(Trees.scala:1306)
```

**Comment by fsanaulla** [comment]
It fails on scalac typer phrase. Can be a part of [this](https://issues.scala-lang.org/browse/SI-9696).

**Comment by fsanaulla** [comment]
Adding `-Ymacro-debug-lite`, flag shows that compiler falls to infinity macro expansion loop.

**Comment by fsanaulla** [comment]
Solved in #47.
